### ISSUE TITLE
Simplify CI build environment

### ISSUE THREAD

**Issue by randyzwitch** [omnisci/pymapd#149]
With #148, build environment shouldn't need anything special beyond common Python dependencies. Consider simplifying CI environment to remove conda, so that just base Travis functionality used. This will allow us to continue to verify that pymapd builds correctly via pip, as well as allow for using more of the functionality provided by Travis around versions, allowable failures, etc.

Alternatively with #134, we could consider moving to a single build environment that supports both Windows and Linux/OSX.

**Comment by randyzwitch** [comment]
#240 is work towards this PR, switching from Travis to Jenkins

@mattdlh Jenkins is just set up for linux now? Is there opportunity for testing on Windows (and OSX, if it isn't annoying).

**Comment by mattdlh** [comment]
Yes we are just testing on linux now. Since we are using our open-source Jenkins to run these, we do not currently have any Windows or OSX hosts associated with it for builds; though both are possible. Windows would certainly be the easier of the two to implement.

**Comment by randyzwitch** [comment]
Thanks. Windows is the one I'm more concerned with, since we've got plenty of OSX users to report when things don't work
